web_tree_dynamic_colored_field/demo/res_users.xml

```diff
@@ -0,0 +1,22 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<openerp>
+    <data>
+        <record id="view_users_tree" model="ir.ui.view">
+            <field name="model">res.users</field>
+            <field name="inherit_id" ref="base.view_users_tree" />
+            <field name="arch" type="xml">
+                <xpath expr="." position="attributes">
+                    <attribute name="color_field">lang</attribute>
```

Comment:
Is this a valid example? I cannot see how the field 'lang' of the user can change the color of the record. For example, if the language is "EN"(English).